src/lib.rs

```diff
@@ -1,16 +1,17 @@
 #![no_std]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
-//! Interoperability library for Rust Windowing applications.
+//! Interoperability library for Rust Windowing applications and Rust applications that connect
+//! to system display server.
```

Comment:
```suggestion
//! Interoperability library for Rust Windowing applications.
```

Leaving this as-is might be fine, as this is still the primary purpose of `raw-window-handle`.